src/parquet/reader.cc

```diff
@@ -217,6 +217,9 @@ static string parquet_type_to_string(Type::type t) {
     case Type::INT64:
       return "INT64";
       break;
+    case Type::INT96:
+      return "INT96";
```

Comment:
Aside: we should move these names to `static constexpr const char*` members of `type_traits`